src/azure-cli/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -196,7 +196,7 @@ def _get_aad_token_after_challenge(cli_ctx,
     }
 
     logger.debug(add_timestamp("Sending a HTTP Post request to {}".format(authhost)))
-    response = requests.post(authhost, urlencode(content), headers=headers,
+    response = requests.post(url=authhost, data=urlencode(content), headers=headers,
                              verify=not should_disable_connection_verify())
```

Comment:
I see this same request being used on line 303 as well....to maintain consistency and avoid potential issues with positional arguments, should we also change line 303 to use keyword arguments?